components/tag-list/tag-list.js

```diff
@@ -193,18 +192,17 @@ class TagList extends LocalizeCoreElement(InteractiveMixin(ArrowKeysMixin(LitEle
 			<div
 				aria-label="${this.description}"
 				class="${classMap(containerClasses)}"
-				role="list"
+				role="group"
```

Comment:
Since we are moving focus internally, we lose the list-specific announcements that were announced with NVDA. To make the semantics clearer, we can use the `group` role and provide a custom description for the role. Should we choose to, we could append counts to the description, but these they were not announced consistently across browsers+screen readers previously anyway.